.github/actions/activate-emsdk/action.yml

```diff
@@ -21,4 +21,6 @@ runs:
       run: |
         cd build/external/emsdk-${{ inputs.version }}/emsdk/ && \
         ./emsdk install ${{ inputs.version }} && \
-        ./emsdk activate ${{ inputs.version }}
\ No newline at end of file
+        ./emsdk activate ${{ inputs.version }} && \
+        echo "${{github.workspace}}/build/external/emsdk-${{ inputs.version }}/emsdk/" >> $GITHUB_PATH && \
+        echo "${{github.workspace}}/build/external/emsdk-${{ inputs.version }}/emsdk/upstream/emscripten" >> $GITHUB_PATH
```

Comment:
Is it reasonable to pull this out into a separate step? `- name: Add emscripten to GitHub PATH` or something like that?